src/design-system/asset-input/simple-asset-input.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { AmountInput } from './amount-input.component';
+import { AmountInputAlignments } from './amount-input.css';
+import * as cx from './simple-asset-input.css';
+
+import type { Asset, AssetState } from './asset-input.data';
+
+interface Props {
+  state: AssetState;
+  translations: {
+    balance: string;
+  };
```

Comment:
We struggle with unnecessary re renders, so it would be good to find other option. In that specific case if we deal with only one translation maybe it's ok to just add as prop